go/store/types/map_leaf_sequence.go

```diff
@@ -194,41 +200,51 @@ func (ml mapLeafSequence) entries() (mapEntrySlice, error) {
 }
 
 func (ml mapLeafSequence) getCompareFn(other sequence) compareFn {
```

Comment:
the diff git shows of this function sucks.  The existing logic is unchanged but put into the else block.  The if block is new.